patroni/dcs/__init__.py

```diff
@@ -458,9 +490,20 @@ def cancel_initialization(self):
         """ Removes the initialize key for a cluster """
 
     @abc.abstractmethod
-    def delete_cluster(self):
+    def delete_cluster(self, index=None):
```

Comment:
I am fine with it, but please also fix https://github.com/zalando/patroni/pull/314/files#diff-0996184a693191991bd4f56d522c4f32R50 and https://github.com/zalando/patroni/pull/314/files#diff-8ac0f76acf09bfec37a70fed766b3d27R326

Also `setup.py flake8` complains about `E226 missing whitespace around arithmetic operator`
Probably it is because: https://github.com/zalando/patroni/pull/314/files#diff-b91f3d5bd63fcd17221b267e851608e8R3

Somehow flake8 doesn't like comment lines, probably because they start not from begging of the string. And actually after removing comments result is absolutely the same as not having `ignore` at all.